### ISSUE TITLE
[Pricing Table]: QA: When the button is expanded, icon is changed to 'X' instead of up arrow.

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#9160]
### Summary

Issue: When the button is expanded, icon is changed to 'X' instead of up arrow.

Expected: When the button is expanded, down arrow icon is shown in the button. Similarly when it is collapsed, up arrow icon should be shown in the button.

Component: Pricing Table

Version: v1.36.0-rc.0

Package
@carbon/ibmdotcom-web-components

CodeSandbox example
https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components-experimental

### Name and organization

Name: Gopinathan
Organization: IBM

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by proeung** [comment]
@Gopi916 I believe this is by design, but I'll defer to @jwitkowski79 and @oliviaflory on this.
